d2cli/main.go

```diff
@@ -76,6 +77,10 @@ func Run(ctx context.Context, ms *xmain.State) (err error) {
 	if err != nil {
 		return err
 	}
+	animationIntervalFlag, err := ms.Opts.Int64("D2_ANIMATION_INTERVAL", "animation-interval", "", -1, "if given, multiple boards are packaged as 1 SVG which transitions through each board at the given interval (in milliseconds)")
```

Comment:
@nhooyr re our convo about animating steps, i'm thinking of just using this flag to control

Reply:
maybe shorten it to `animate-interval`?